ibis/pyspark/compiler.py

```diff
@@ -280,6 +283,188 @@ def compile_greatest(t, expr):
         return F.greatest(*src_columns)
 
 
+@compiles(ops.Least)
+def compile_least(t, expr):
+    op = expr.op()
+
+    src_columns = t.translate(op.arg)
+    if len(src_columns) == 1:
+        return src_columns[0]
+    else:
+        return F.least(*src_columns)
+
+
+@compiles(ops.Abs)
+def compile_abs(t, expr):
+    op = expr.op()
+
+    src_column = t.translate(op.arg)
+    return F.abs(src_column)
+
+
+@compiles(ops.Round)
+def compile_round(t, expr):
+    op = expr.op()
+
+    src_column = t.translate(op.arg)
+    scale = op.digits.op().value if op.digits is not None else 0
+    rounded = F.round(src_column, scale=scale)
+    if scale == 0:
+        rounded = rounded.astype('long')
+    return rounded
+
+
+@compiles(ops.Ceil)
+def compile_ceil(t, expr):
+    op = expr.op()
+
+    src_column = t.translate(op.arg)
+    return F.ceil(src_column)
+
+
+@compiles(ops.Floor)
+def compile_floor(t, expr):
+    op = expr.op()
+
+    src_column = t.translate(op.arg)
+    return F.floor(src_column)
+
+
+@compiles(ops.Exp)
+def compile_exp(t, expr):
+    op = expr.op()
+
+    src_column = t.translate(op.arg)
+    return F.exp(src_column)
+
+
+@compiles(ops.Sign)
+def compile_sign(t, expr):
+    import numpy as np
+    op = expr.op()
+
+    @pandas_udf('double', PandasUDFType.SCALAR)
```

Comment:
Can we rewrite this using sql operations? e.g

```
F.when(src_column == 0, lit(0)).otherwise(F.when(src_column > 0, lit(1)).otherwise(-1))
```

Reply:
Done